src/runtime.jl

```diff
@@ -108,6 +108,10 @@ function interpolate_ast(ex, values...)
     end
 end
 
+function interpolate_ast(::Type{Expr}, ex, values...)
+    Base.Expr(interpolate_ast(SyntaxTree, ex, values...))
```

Comment:
```suggestion
    Expr(interpolate_ast(SyntaxTree, ex, values...))
```
`Expr` is exported from Core, so we don't need to use the explicit `Base` accessor even in this `baremodule` context.